kof-operator/internal/controller/remote-secret/remote_secret_manager.go

```diff
@@ -0,0 +1,166 @@
+package remotesecret
+
+import (
+	"context"
+	"fmt"
+
+	kcmv1alpha1 "github.com/K0rdent/kcm/api/v1alpha1"
+	"github.com/go-logr/logr"
+	istio "github.com/k0rdent/kof/kof-operator/internal/controller/isito"
+	"istio.io/istio/istioctl/pkg/multicluster"
+	"istio.io/istio/pkg/kube"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const RemoteSecretNamespace = "istio-system"
```

Comment:
We have similar const
```
const istioCANamespace = "istio-system"
```
Let's combine them into a single const IstioSystemNamespace